src/plot/axisannotation.js

```diff
@@ -138,31 +138,19 @@ function backgroundPath(accessor, axis, height, width, point, neg) {
         if(height/2 < point) pt = height/2;
         else h = height/2-point;
 
-        return [
-          'M', 0, value,
-          'l', neg*Math.max(axis.innerTickSize(), 1), -pt,
-          'l', 0, -h,
-          'l', neg*width, 0,
-          'l', 0, height,
-          'l', neg*-width, 0,
-          'l', 0, -h
-        ].join(' ');
+        return 'M 0 ' + value + ' l ' + String(neg*Math.max(axis.innerTickSize(), 1)) + ' ' + String(-pt) +
```

Comment:
Another great catch. Those `join`s in chrome are a killer.
Any reason for `String` constructor rather than implicit conversion? For my profiling in chrome implicit conversion is much faster. Will post cases for review.